docs/en/09-data-out/_sub_cs.mdx

```diff
@@ -1,3 +1,3 @@
 ```csharp
-// {{#include docs/examples/csharp/SubscribeDemo.cs}}
+{{#include docs/examples/csharp/native-example/SubscribeDemo.cs}}
```

Comment:
Suggest to put this line in the 01-query-data.md directly, and emove this _sub_cs.mdx, to make it easier to maintain.

Reply:
ok